proc/groupby/ztests/dot.yaml

```diff
@@ -1,6 +1,6 @@
 zql: 'count() by typeof(.) | sort count'
 
-output-flags: -f zson
+output-flags: -pretty=4
```

Comment:
Should this be `-f zson -pretty=4`?

Reply:
Just pushed a fix.  -f zson is the proposed default so you don't need it here but I added for clarity and consistency.